tests/bearlib/aspects/ClassTest.py

```diff
@@ -31,3 +33,9 @@ class docs:
                 example = 'Example'
 
         assert not SubAspect.docs.check_consistency()
+        assert issubaspect(RootAspect, SubAspect)
+        assert not issubaspect(RootAspect, Root)
+        assert not issubaspect(RootAspect, RootAspect)
+        with pytest.raises(TypeError):
+            issubaspect('String', SubAspect)
+            issubaspect(RootAspect, 'String')
```

Comment:
You should also check other class objects here and not only instance objects of other classes. So for example also `str`.